.github/workflows/nim-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Nim Compilability CI
+
+on:
+  push:
+    paths:
+      - "**.nim"
+  pull_request:
+    paths:
+      - "**.nim"
```

Comment:
My thinking was that the only nim files would be for exercises so to avoid having to change this if there were changes to the repo structure, I assumed that `**.nim` would be ok.

I'm happy to commit this suggestion if you'd like. Doesn't make much of a difference to me.

Reply:
I'd prefer the explicit version.